test/greet-test.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const chai = require('chai');
+const greet = require('../lib/greet.js');
+
+const expect = chai.expect;
+
```

Comment:
Remember to also test that your argument is a string if you want to make changes and resubmit for points.